public/configs/env.json

```diff
@@ -186,7 +187,8 @@
             "strideZoneApp": "https://testnet.stride.zone",
             "accountExportLearnMore": "https://help.dydx.exchange/en/articles/8565867-secret-phrase-on-dydx-chain",
             "walletLearnMore": "https://www.dydx.academy/video/defi-wallet",
-            "withdrawalGateLearnMore": "https://help.dydx.exchange/en/articles/8981384-withdrawals-on-dydx-chain#h_23e97bc665"
+            "withdrawalGateLearnMore": "https://help.dydx.exchange/en/articles/8981384-withdrawals-on-dydx-chain#h_23e97bc665",
+            "exchangeStats": "https://app.mode.com/dydx_eng/reports/58822121650d?secret_key=391d9214fe6aefec35b7d35c"
```

Comment:
there's a secret_key here, is this okay?

Reply:
Yep, for whatever reason Mode spits out a URL with that "secretKey" param for externally shared URLs.